CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## v0.38.15
+
+*November 6, 2024*
+
+This release supersedes [`v0.38.14`](#v03814), which mistakenly updated the Go version to
+`1.23`, introducing an unintended breaking change. It set the Go version back
```

Comment:
```suggestion
`1.23`, introducing an unintended breaking change. It sets the Go version back
```
Although I guess this is generated by unclog automatically?

Reply:
yes